### PR TITLE
add karmada commercial support template

### PR DESCRIPTION
**Pull request by wawa0210**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
I created a repo for testing and the effect is as follows

image

image

The final issues : https://github.com/wawa0210/karmada-commercial-demo/issues/1

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by RainbowMango** [comment]
Can you put it in another repo(like a personal one) so we can test it there?

**Comment by wawa0210** [comment]
> Can you put it in another repo(like a personal one) so we can test it there?

Already added to pr description，thx

**Review by RainbowMango — COMMENTED** [review]
/assign

**Review by RainbowMango — COMMENTED** [review]
Generally looks good to me.

I think we can iterate it later once we have more description of the commercial support.

File: .github/ISSUE_TEMPLATE/commercial-support-partner.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Karmada Commercial Support Request
+description: Request to be a Karmada Commercial Support Partner
+title: "REQUEST: Organization <Organization name> that provides commercial support to karmada New Commercial Support "
```
[the remark this answers is not in the document]
Comment (wawa0210) [reply .github/ISSUE_TEMPLATE/commercial-support-partner.yml:3]: maybe `area/commercial-support` ?

I don’t have permission, can you help create it?

File: .github/ISSUE_TEMPLATE/commercial-support-partner.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- id: org_logo
+  type: textarea
+  attributes:
+    label: "Org or company logo"
+    description: Organizations offering business support programs logo images(perfer png)
+- id: contact
+  type: textarea
+  attributes:
+    label: "Contact me"
+    description: Contact information of the organization that provides commercial support(github id or email)
```
[the remark this answers is not in the document]
Comment (wawa0210) [reply .github/ISSUE_TEMPLATE/commercial-support-partner.yml:29]: image
it shows good

File: .github/ISSUE_TEMPLATE/commercial-support-partner.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (RainbowMango) [reply .github/ISSUE_TEMPLATE/commercial-support-partner.yml]: Sure, I'll add it. Maybe after this PR.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**